### ISSUE TITLE
Received a label value of 1 which is outside the valid range error

### ISSUE THREAD

**Issue by kristiawannugroho** [bagustris/ravdess_ser#1]
I managed to run this script successfully using the RAVDES SER dataset 
but when I use my own dataset at the time the model training error occurs at
invalidargumentError error: Received a label value of 1 which is outside the valid range of [0, 1).  Label values: 1 1
[[node loss/dense_5_loss/sparse_categorical_crossentropy/SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits 
(defined at /usr/local/lib/python3.6/dist-packages/tensorflow_core/python/framework/ops.py:1751) ]] [Op:__inference_keras_scratch_graph_1668]

I beg to be given a solution to my problem, thank you.

**Comment by bagustris** [comment]
Dear @kristiawannugroho

I suppose this error due to labeling schema.

In the given example, the label is arranged as this (file `y.npy`):
```
array([[1., 0., 0., ..., 0., 0., 0.],
       [1., 0., 0., ..., 0., 0., 0.],
       [1., 0., 0., ..., 0., 0., 0.],
       ...,
       [0., 0., 1., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 1., 0.]], dtype=float32)
```

If your label in ordinal format, e.g., [1, 2, 1, 3, 1, 4, 1, 3, 4, 2, 1, ...], there is an option available: convert the labels into binary using one-hot encoding (using scikit-learn's "LabelBinarizer" or Keras's to_categorical)

Another option is by using different loss functions than categorical_crossetropy. But the previous solution is preferable.

Check again your label if you think you already binarize it.

**Comment by kristiawannugroho** [comment]
Okay nice ... thank you for your solution, the script can work properly now
